cmp/cmpopts/sort.go

```diff
@@ -94,7 +94,8 @@ func (ss sliceSorter) less(v reflect.Value, i, j int) bool {
 //	• Transitive: if !less(x, y) and !less(y, z), then !less(x, z)
 //	• Total: if x != y, then either less(x, y) or less(y, x)
 //
-// SortMaps can be used in conjuction with EquateEmpty.
+// SortMaps can be used in conjunction with EquateEmpty,
+// but cannot be used with DiscardMapZeros.
```

Comment:
> Perhaps this is another good use-case for the "first option that applies" combiner you mentioned at GopherCon.

Yep. I was thinking the exact same thing :)

It would be really cool to be able to do something like FilterPriority(DiscardMapZeros(), SortMaps(f)) to first discard the map zeros and then sort the map.

Let's leave the comments clean-up for the CL that adds in the priority filter idea.